etna/search/templatetags/search_tags.py

```diff
@@ -84,17 +84,6 @@ def include_hidden_fields(visible_field_names, form) -> str:
     return mark_safe(html)
 
 
-@register.simple_tag(takes_context=True)
-def has_result_count(context, bucket) -> bool:
-    """Returns True if at least one bucket contains count returned by api response, otherwise False"""
-    buckets = context["buckets"]
-    for bucket in buckets:
-        if bucket.result_count:  # checks api has returned a count
-            if bucket.result_count > 0:
-                return True
-    return False
```

Comment:
The functionality to check  least one bucket is removed. What is that replaced with?

Reply:
It is no longer needed because we add `buckets_contain_results` to the template context